content/pt-br/docs/reference/glossary/shuffle-sharding.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Shuffle-sharding
+id: shuffle-sharding
+date: 2020-03-04
+full_link:
+short_description: >
+  Uma técnica para atribuir requisições para filas que proporciona melhor isolamento do que hashing módulo do número de filas.
+
+aka:
+tags:
+- fundamental
+---
+Uma técnica para atribuir requisições para filas que proporciona melhor isolamento do que hashing módulo do número de filas.
+
+<!--more-->
+
+Nos preocupamos frequentemente em isolar diferentes fluxos de requisições
+uma das outras, de modo que um fluxo de alta intensidade não afete um fluxo de baixa intensidade.
+Uma forma simples de colocar requisições em fila é gerar um hash baseado
+em características da requisição, módulo do número de filas, para ter o
```

Comment:
```suggestion
em características da requisição, efetuar a operação módulo, ou resto da divisão, do _hash_ calculado pelo número de filas, para ter o
```

Just a suggestion, feel free to word it differently if you feel otherwise.